### PR TITLE
feat(contribution-type): added maintenance type

### LINKED ISSUE #111: New Category:  maintenance
Love the tool.  I was thinking it might be great if we could have a new category for those who simply come along and update dependencies.  They actually don't add code, but they help dependencies, and they maintain the "freshness" of a repo.

I've had a few come do that.

### PR DESCRIPTION
**Pull request by Berkmann18**
fixes #111

**What**:
Added 👷 (maintenance) to the types.

**Why**:
It was suggested by @GantMan in #111 and accepted by @kentcdodds.

**How**:
Addition to `./src/util/contribution-types`.

**Checklist**:

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

**Comment by jakebolam** [comment]
Looks good, I left a comment on #111

**Comment by Berkmann18** [comment]
It's now using 🚧 🎉 !

**Comment by GantMan** [comment]
awesome!!!!!!

File: README.md [hunk 1]
```diff
@@ -145,6 +144,7 @@ Where `username` is the user's GitHub or Gitlab username, and `contribution` is
 - tutorial: [✅](# 'Tutorials')
 - userTesting: [📓](# 'User Testing')
 - video: [📹](# 'Videos')
+- maintenance [🚧](# 'Maintenance')
```
Comment (jakebolam) [inline README.md]: Could we maintain alphabetical order?

Comment (Berkmann18) [reply README.md]: Sure.

File: src/util/contribution-types.js [hunk 2]
```diff
@@ -99,6 +99,10 @@ const defaultTypes = function(repoType) {
       symbol: '📹',
       description: 'Videos',
     },
+    maintenance: {
```
[the remark this answers is not in the document]
Comment (Berkmann18) [reply src/util/contribution-types.js]: Good point, I'll get on it now.

**Comment by Berkmann18** [comment]
✔️ Alphabetical order
✔️ spec updated (cf [PR](https://github.com/all-contributors/all-contributors/pull/126) above)
✔️ website updated (cf [PR](https://github.com/all-contributors/website/pull/6) above).

**Pull request merged**